docs/gatsby-ssr.js

```diff
@@ -0,0 +1,27 @@
+// Import React so that you can use JSX in HeadComponents
+const React = require('react')
+
+const HtmlAttributes = {
+  lang: 'en'
+}
+
+const HeadComponents = [
+  <link
+    href="https://cloud.typography.com/6966154/6397212/css/fonts.css"
+    key="font-import"
+    referrerPolicy="strict-origin"
```

Comment:
this referrer policy should allow the font to load even with our more robust helmet csp settings 🤞